### ISSUE TITLE
TXMLReader never used

### ISSUE THREAD

**Issue by oscargus** [koppor/jabref#25]
While browsing the code, I realized that the TXMLReader is never used. Or rather, it is used as a possible constructor in BibtexSingleField (in BibtexFields):
``` java
        /**
         * the constructor reads all neccessary data from the xml file
         */
        public BibtexSingleField(TXMLReader reader, Element node) {
```
which in turn is never used.

Remove?

**Comment by koppor** [comment]
Oh, well. This could have been used in the importers / fetchers. Either go through *all* of them and check whether the code can be tweaked to use TXMLReader (which could make sense to avoid code duplication) or simply remove it.

I know that JAX-B is the way better mechanism than interpreting XML manually. However, when seeing GVB files (e.g., https://github.com/JabRef/jabref/blob/gvk_fetcher/src/test/resources/net/sf/jabref/importer/fetcher/gvk_gmp.xml) then I assume not everyone has understood the concept of XML elements and thus fosters the need for hand-crafted parsers.

**Comment by simonharrer** [comment]
:+1: for removing code.

**Comment by oscargus** [comment]
Closed with JabRef/jabref#392
